### PR TITLE
Make multiselects full-width for better look-and-feel

### PR DESCRIPTION
**Pull request by ejjoman**
* Make multiselects full-width for better look-and-feel
* Center the exchange icon between those two lists
* Fix: Change exchange icon from **\<li\>** to **\<span\>** because \<li\> needs \<ul\> or \<ol\> as parent

### CODE REVIEW

**Comment by nijel** [comment]
The icon was originally <i>, not <li>, so it should be perfectly valid there...

File: weblate/static/multi-select.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .ms-container{
   background: transparent url('../img/switch.png') no-repeat 50% 50%;
-  width: 370px;
+  width: 100%;
```
Comment (nijel) [inline weblate/static/multi-select.css]: Please make the change rather in style-bootstrap.css. This file is taken verbatim from multi-select upstream and will be replaced on possible upgrades.

File: weblate/static/loader-bootstrap.js [hunk 2]
```diff
@@ -751,7 +751,7 @@ $(function () {
         $(this).multiSelect({
             afterInit: function (target) {
                 $(target.children()[0]).after(
-                    '<div class="fa-multiselect"><i class="fa fa-exchange"></i></div>'
+                    '<div class="fa-multiselect"><span class="fa fa-exchange"></span></div>'
```
Comment (nijel) [inline weblate/static/loader-bootstrap.js]: I don't see what is wrong on using <i> here....

Comment (ejjoman) [inline weblate/static/loader-bootstrap.js]: Sorry, it was late last night :) somehow I read "li" instead of "i"...

**Comment by nijel** [comment]
I've made the changes in a661389, thanks for suggestions.

**Pull request closed without merging**